Frontend/ui-customer/index.js

```diff
@@ -132,7 +132,10 @@ window.onMerchantEvent = function (_event, merchantPayload) {
       merchant = merchant.replace("consumer","")
       merchant = merchant.toUpperCase();
       window.merchantID = "MOBILITY_" + merchant.charAt(0) + merchant.charAt(merchant.length - 1);
-    } else if (clientId.includes("consumer")) {
+    } else if (clientId.includes("redbus")) {
```

Comment:
add it in before case itself `else if (clientId.includes("mobility") || clientId.includes("redbus"))` like this

Reply:
```
if (clientId.includes("_ios"))
    {
      clientId = clientId.replace("_ios","");
    }
    if (clientId == "open-kochi" || clientId == "yatriconsumer") {
      window.merchantID = "YATRI"
    } else if (clientId == "jatrisaathi" || clientId == "jatrisaathiconsumer"){
      window.merchantID = "YATRISATHI"
    }  else if (clientId.includes("consumer")) {
      let merchant = clientId.replace("mobility","")
      merchant = merchant.replace("consumer","")
      window.merchantID = merchant.toUpperCase();
    } else {
      let merchant = clientId.replace("mobility","")
      merchant = merchant.replace("consumer","")
      merchant = merchant.toUpperCase();
      window.merchantID = "MOBILITY_" + merchant.charAt(0) + merchant.charAt(merchant.length - 1);
    }
```

update the whole block like this